### ISSUE TITLE
Tree - Context menu can't be opened using keyboard

### ISSUE THREAD

**Issue by infe42** [primefaces/primeng#19554]
### Describe the bug

The context menu of a tree item can't be opened using the keyboard (context menu key or context menu shortcut), because the `(contextmenu)` event listener is attached to the wrong element - the `<li>` is focused, but the event listener is on the `<div>` inside it.

### Pull Request Link

https://github.com/primefaces/primeng/pull/18254

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/tree#contextmenu

### Environment

-

### Angular version

21.x.x

### PrimeNG version

v21

### Browser(s)

all

### Steps to reproduce the behavior

1. Go to https://primeng.org/tree#contextmenu
2. Click a tree item
3. Press the context menu key or the context menu shortcut (Shift + F10 in Chrome)
Actual behavior: The browser context menu is opened instead of the tree context menu.

### Expected behavior

The tree context menu is opened.
